src/messaging/outgoing.ts

```diff
@@ -4,6 +4,7 @@ export const EXIT_FULLSCREEN = "exit-fullscreen";
 
 export const INITIALIZE_REQUEST = "initialize-request";
 export const RESET_IFRAME_REQUEST = "reset-iframe-request";
+export const LOGIN_HANDLER_REGISTERED = "login_handler_registered";
```

Comment:
Any reason for us to propagate this to core? Dont think we have any implementation to handle that.

Reply:
No reason, I was just testing and forgot to remove